src/tests/unit/tests/DetailsView/components/medium-pass-command-bar.test.tsx

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import {
+    CommandBarProps,
+    DetailsViewCommandBarProps,
+} from 'DetailsView/components/details-view-command-bar';
+import { DetailsViewSwitcherNavConfiguration } from 'DetailsView/components/details-view-switcher-nav';
+import { MediumPassCommandBar } from 'DetailsView/components/medium-pass-command-bar';
+import { shallow } from 'enzyme';
+import * as React from 'react';
+
+function getProps(): CommandBarProps {
+    const switcherNavConfiguration: DetailsViewSwitcherNavConfiguration = {
+        ReportExportComponentFactory: (p: DetailsViewCommandBarProps) => null,
+        StartOverComponentFactory: (p: DetailsViewCommandBarProps) => null,
+    } as unknown as DetailsViewSwitcherNavConfiguration;
```

Comment:
nit: is the `as unknown` required? I've noticed that as long as your types match the intended type you're trying to cast, the `as SOME-TYPE` is enough.